docs/differential-analysis.rst

```diff
@@ -0,0 +1,165 @@
+You can read more about it `here <https://developer.carbonblack.com/reference/carbon-black-cloud/cb-liveops/latest/differential-analysis-api/#query-comparison>`_.
+
+Query Comparison
+----------------
+
+Basic Query
+###########
+
+This example shows the basic result of the ``Differential`` object. The ``.newer_run_id()`` method is required - it accepts the
+run id that you want to mark as the starting point-in-time snapshot. By default, only the number of changes between the two runs are returned.
+To receive the actual differential data, use the ``.count_only()`` method, as featured in the Actual Changes example.
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.audit_remediation import Differential
+    >>>
+    >>> cbc = CBCloudAPI(profile='sample')
+    >>>
+    >>> resp = cbc.select(Differential).newer_run_id('jcdqsju4utpaayj5dh5r2llzffeolg0u').older_run_id('yhbg3wcea9y1l4asiltky5tupkgauzas')
+    >>> # The content of resp object always has a length of 1, and contains the Differential response.
+    >>> print(*resp)
+    Differential object, bound to https://defense-dev01.cbdtest.io.
+    diff_processed_time: 0.633
+    diff_results: [list:1 item]:
+    [0]: {'device_id': 12345, 'change_count': 1, 'add...
+    newer_run_create_time: 2022-08-10T13:07:44.194Z
+    newer_run_id: jcdqsju4utpaayj5dh5r2llzffeolg0u
+    newer_run_not_responded_devices: [list:0 items]
+    older_run_create_time: 2022-08-10T12:57:03.872Z
+    older_run_id: yhbg3wcea9y1l4asiltky5tupkgauzas
+    older_run_not_responded_devices: [list:0 items]
+
+
+You can also access a dictionary representation of the response with the ``._info`` property.
+
+    >>> print(resp[0]._info)
+    {'diff_processed_time': 0.037,
+     'diff_results': [{'added_count': 1,
+                       'change_count': 1,
+                       'changes': None,
+                       'device_id': 12345,
+                       'newer_run_row_count': 21,
+                       'older_run_row_count': 20,
+                       'removed_count': 0}],
+     'newer_run_create_time': '2022-08-10T13:07:44.194Z',
+     'newer_run_id': 'jcdqsju4utpaayj5dh5r2llzffeolg0u',
+     'newer_run_not_responded_devices': [],
+     'older_run_create_time': '2022-08-10T12:57:03.872Z',
+     'older_run_id': 'yhbg3wcea9y1l4asiltky5tupkgauzas',
+     'older_run_not_responded_devices': []}
+
+
+Actual Changes
+##############
+
+Using the ``.count_only()`` method with a value of ``False`` will allow you to see the actual changes between the two snapshots.
+To use this method, append it to the rest of the Differential object query. The actual changes will be in the ``changes`` property, under ``diff_results``.
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.audit_remediation import Differential
+    >>>
+    >>> cbc = CBCloudAPI(profile='sample')
+    >>>
+    >>> resp = cbc.select(Differential).newer_run_id('jcdqsju4utpaayj5dh5r2llzffeolg0u').older_run_id('yhbg3wcea9y1l4asiltky5tupkgauzas')
```

Comment:
Does this actually make an API call? or is the `resp` variable still the query object? If the variable is still the query object then we should call it `query`

Reply:
Please apply the same logic to all examples